packages/atlas/src/views/studio/PlaylistWorkspace/PlaylistWorkspace.tsx

```diff
@@ -0,0 +1,299 @@
+import { useCallback, useEffect, useState } from 'react'
+import { Controller, useForm } from 'react-hook-form'
+
+import { useBasicVideo } from '@/api/hooks/video'
+import { SvgActionAdd } from '@/assets/icons'
+import { DraggableComponent } from '@/components/DraggableComponent/DraggableComponent'
+import { EmptyFallback } from '@/components/EmptyFallback'
+import { Button } from '@/components/_buttons/Button'
+import { FormField } from '@/components/_inputs/FormField'
+import { ImageUploadAndCrop } from '@/components/_inputs/ImageUploadAndCrop/ImageUploadAndCrop'
+import { ImageInputFile, ImageInputMetadata } from '@/components/_inputs/MultiFileSelect'
+import { OptionCardGroupRadio } from '@/components/_inputs/OptionCardGroup'
+import { TextArea } from '@/components/_inputs/TextArea'
+import { TitleInput } from '@/components/_inputs/TitleInput'
+import { BottomDrawer } from '@/components/_overlays/BottomDrawer'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+import { useAsset, useRawAsset } from '@/providers/assets/assets.hooks'
+import { useAssetStore } from '@/providers/assets/assets.store'
+import { createId } from '@/utils/createId'
+import { computeFileHash } from '@/utils/hashing'
+import { SentryLogger } from '@/utils/logs'
+import { VideoSelectorDialog } from '@/views/studio/PlaylistWorkspace/VideoSelectorDialog/VideoSelectorDialog'
+
+import { FormWrapper, StyledButton, StyledVideoListItem, WorkspaceWrapper } from './PlaylistWorkspace.styles'
+
+type PlaylistWorkspaceFormFields = {
+  title: string
+  description: string
+  isPublic: boolean
+  thumbnail: {
+    cropId: string | null
+    originalId: string | null
+    originalBlob?: {
+      name?: string
+    } | null
+  } & ImageInputMetadata
+}
+
+const formOptions = [
+  {
+    value: true,
+    label: 'Public',
+    caption: 'Visible to all',
+  },
+  {
+    value: false,
+    label: 'Unlisted',
+    caption: 'Visible only with link',
+  },
+]
+// todo: should be transfered to the context later on
+export type PlaylistWorkspaceProps = {
+  show: boolean
+  onHide: () => void
+}
+
+export const PlaylistWorkspace = ({ show, onHide }: PlaylistWorkspaceProps) => {
```

Comment:
I guess we forgot to mention that, but we're used to using `FC` for all our components i.e `export const Component: FC<ComponentProps>`. Can we use it for the sake of consistency?